qds_sdk/commands.py

```diff
@@ -87,10 +87,26 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
             time.sleep(Qubole.poll_interval)
             cmd = cls.find(cmd.id)
+            if kwargs.get('print_logs_live', False):
+                log, err_length, tmp_length = cmd.get_log_partial(err_pointer, tmp_pointer)
+
+                if err_length != "0":
+                    err_pointer += int(err_length)
+                    new_bytes = int(err_length) + int(tmp_length) - new_bytes - tmp_pointer
```

Comment:
Yes, this is largely meant for composite command. This logic is helpful in more case, command might have finished when you called logs api, so it may return non-empty err logs and empty tmp logs.